ee/agent/storage/sqlite/migrations/000001_kv.up.sqlite

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS keyvalue_pairs (
```

Comment:
In my head, I think of the KV store as sqlite:table::boltdb:bucket. In that vein, this should probably be named something else. (probably also true up in the go code)

Reply:
Do you have suggestions for another name? I've renamed this a couple times and haven't landed on anything that seemed great. It was `startup_flags` before, which was too specific.

Reply:
`startup_settings` isn't bad.

Reply:
I like that!